cumulusci/cli/ui.py

```diff
@@ -133,3 +133,137 @@ def _dim_value(self, val):
         # If a string has been wrapped, wrap each line to avoid dimming table borders
         dimmed_strs = [click.style(line, dim=True) for line in val.split("\n")]
         return "\n".join(dimmed_strs)
+
+
+def pretty_soql_query(sf, query, format, include_deleted=False, max_rows=100):
+    """Return the result of a Salesforce SOQL query.
+
+    Arguments:
+
+        * query -- the SOQL query to send to Salesforce, e.g.
+                   SELECT Id FROM Lead WHERE Email = "waldo@somewhere.com"
+        * include_deleted -- True if deleted records should be included
+        * format -- one of these values:
+           - "table" -- printable ASCII table (the default)
+           - "obj" -- ordinary Python objects
+           - "pprint" -- string in easily readable Python dict shape
+           - "json" -- JSON
+        * max_rows -- maximum rows to output, defaults to 100
+
+    For example:
+        pretty_soql_query("select Name, Id from Account", format="pprint")
+        contact_ids = pretty_soql_query("select count(Id) from Contact", format="obj")
+    """
+    results = sf.query(query, include_deleted=include_deleted)["records"]
```

Comment:
Careful -- `query` only returns the first batch of results (200 I think). In order to get the rest you have to do additional requests using `query_more`.